lists/US-SAM

```diff
@@ -0,0 +1,49 @@
+{
+  "name": {
+    "en": "System for Award Management",
+    "local": ""
+  },
+  "url": "https://sam.gov/content/entity-registration",
+  "description": {
+    "en": "The System for Award Management (SAM) of U.S. General Services Administration (GSA). The System for Award Management website (SAM.gov) is an official website of the U.S. Government that allows users to register to do business with the U.S. Government\nTo do so, and to use other functionality such as full search, requires a login through the US government's Login.gov service. https://www.login.gov/policy/rules-of-use/ states that The Login.gov service is operated within the United States, but is accessible globally for public use. "
```

Comment:
The description needs to be edited to focus on the unique entity identifier issued by SAM. So something like:

"The System for Award Management (SAM) is....blah blah blah

SAM.gov issues the Unique Entity ID which is used across the federal government. (Previously a DUNS Number was used...blah blah"

Reply:
I've made some edits to the description as per your suggestions.